.github/workflows/test-package.yml

```diff
@@ -0,0 +1,107 @@
+name: Test Package (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      package:
+        type: string
+        description: The package to test
+        required: true
+      stylelint-version:
+        type: string
+        description: The Stylelint version to test
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
```

Comment:
[ask] Just curious, is the `strategy` still necessary?

Reply:
Good catch!